client/my-sites/plans-grid/components/actions.tsx

```diff
@@ -131,6 +135,26 @@ const SignupFlowPlanFeatureActionButton = ( {
 		);
 	}
 
+	if (
+		! hasFreeTrialPlan &&
+		isBusinessPlan( planSlug ) &&
+		! isEligibleForTrial &&
+		window.location.pathname.indexOf( 'new-hosted-site' ) !== -1
```

Comment:
Is there a better way to check our flowName in this context than using `pathname`?

Reply:
I am not familiar with the use case, but yes, please look up PlansIntent. There should be a respective one defined for the flow you are working on. We prefer that over flowName also.

Happy to help further on Monday first thing.

Reply:
Excellent, I'll use PlansIntent as well. Thanks, folks!